src/flow.h

```diff
@@ -480,6 +481,9 @@ typedef struct Flow_
      *  has been set. */
     const struct SigGroupHead_ *sgh_toserver;
 
+    /** which exception policy was applied, if any */
```

Comment:
by putting this between 2 pointers we're essentially use 8 bytes for this, which is a lot for something from which we just use 6 bits

Reply:
Sorry, I didn't realize that. >__<'